scripts/lib.sh

```diff
@@ -41,7 +41,7 @@ get_from_kata_deps() {
 	# through CI. For the kernel, .ci/install_kata_kernel.sh file in tests
 	# repository will pass the kernel version as an override to this function to
 	# allow testing of kernels before they land in tree.
-	if ${CI}; then
+	if [ "${CI:-false}" = "true" ]; then
```

Comment:
This needs to change to be the following I think:

```sh
if [ "${CI}" = "true" ]; then
```

Reply:
That change would still leave us with an 'unbound variable' failure when this script is called on non-CI systems. That is what we had before and had to change it to this.

Reply:
In which case it can be:

```sh
if [ "${CI:-}" = "true" ]; then
```